vendor/assets/javascripts/eui-1.0.0/eui.js

```diff
@@ -183,16 +183,7 @@
     });
   });
 }(jQuery));
-;(function () {
```

Comment:
I guess it is too late now, but I don't think this should have been removed. This seems to be the google analytics info from EUI since it is in their JS package, not from MMT. But at the same time, I think we are trying to remove this package to use the more recent version so I guess it is not a big deal.